fotoapparat/src/main/java/io/fotoapparat/hardware/CameraDevice.java

```diff
@@ -49,7 +49,7 @@
     void setDisplaySurface(Object displaySurface);
 
     @Override
-    void setDisplayOrientation(int degrees);
+    void setDisplayOrientation(int degrees, int orientation);
```

Comment:
Since we have 2 parameters now it is not clear anymore which one means what. I suggest to rename first one to `displayOrientationDegrees` and second one to `deviceOrientationDegrees`.

Reply:
I would also prefer to have an `Integer` for `deviceOrientationDegrees` and use `null` instead of using `ORIENTATION_UNKNOWN` from Android. In general, we try to separate our code as much as possible from Android specifics.